mikeio/dfsu.py

```diff
@@ -1082,11 +1082,12 @@ def plot(self, z=None, elements=None, label=None, cmap=None, vmin=None, vmax=Non
             else:
                 # must be contourf plot then
                 levels = np.linspace(vmin, vmax, n_levels)
+                ax.tripcolor(triang, zn, edgecolors='face', vmin=vmin, vmax=vmax, cmap=cmap)
```

Comment:
@mohm-dhi, why do you want to add this line before tricontourf()? What difference does it make?

Reply:
as I explained here https://github.com/DHI/mikeio/pull/76#issue-481622110, without this line when the vmin and vmax are not in the range of the data it creates white area. Please see the figures.

Reply:
right - sorry